docs/self-managed/modeler/desktop-modeler/deploy-to-self-managed.md

```diff
@@ -0,0 +1,33 @@
+---
+id: deploy-to-self-managed
+title: Deploy diagram
```

Comment:
We have then two items in the sidebar:

* Deploy your first diagram (the existing page)
* Deploy diagram (the page you added)

Should the titles not be more verbose so that readers understand from them already what they describe?

Reply:
Do you have any proposal? Note that they don't appear next to each other. Namely, _Deploy your first diagram_ is in Components/Modeler while _Deploy diagram_ is in Self-Managed.

Reply:
Ah - got it.
Sorry, I assumed they appear next to each other. 

In this case: I think this is okay :-)